docs/migration.md

```diff
@@ -27,10 +27,12 @@ At this time, the following ACP-prefix libraries can be switched out with their
 | @adobe/react-native-acpanalytics | NA, workflows supported through [Edge Network](../packages/edge) or [Edge Bridge](../packages/edgebridge) extension. |
 | @adobe/react-native-acpmedia | NA |
 | @adobe/react-native-acpaudience | NA |
-| @adobe/react-native-acptarget | @adobe/react-native-aeptarget |
+| @adobe/react-native-acptarget | @adobe/react-native-aeptarget, for using Analytics for Target, workflows supported through [Edge Network](../packages/edge) and [Optimize](../packages/optimize) extensions. |
```

Comment:
can we split this on two lines, reading:
@adobe/react-native-aeptarget
For using Analytics for Target, workflow is supported through [Edge Network](../packages/edge) and [Optimize](../packages/optimize).